gratipay/models/account_elsewhere.py

```diff
@@ -250,11 +250,16 @@ def save_token(self, token):
 
 
 def get_account_elsewhere(website, state, api_lookup=True):
+    _ = state['_']
     path = state['request'].line.uri.path
+
     platform = getattr(website.platforms, path['platform'], None)
     if platform is None:
         raise Response(404)
+
     uid = path['user_name']
+    if "\t" in uid or "\r" in uid or "\n" in uid:
+        raise Response(400, _("Invalid character in elsewhere account username."))
```

Comment:
Characters would look cleaner here ;)
Btw when do we hit this case and why do we need specifically handle it?